packages/app/src/ResizePlugin.ts

```diff
@@ -9,9 +9,13 @@ import { IApplicationOptions } from './Application';
  */
 export class ResizePlugin
```

Comment:
This plugin is hard to understand with all its static/this context mixed.

Reply:
yeah these are static methods where the context is the Application instance. Loader plugins are similar.